installer/install.iss

```diff
@@ -84,6 +84,7 @@ PrivilegesRequired=lowest
 #else
 PrivilegesRequired=none
 #endif
+UninstallDisplayName={#APP_NAME}
```

Comment:
It is not immediately obvious from the diff why this would fix the issue.

So I had a look what [the description of that directive](https://jrsoftware.org/ishelp/topic_setup_uninstalldisplayname.htm) says:

> If this directive is not specified or is blank, Setup will use the value of `[Setup]` section directive `AppVerName` for the name.

Now, we do not even specify `AppVerName`, therefore I had to look at [that directive's description](https://jrsoftware.org/ishelp/topic_setup_appvername.htm):

> **Default value**: [AppName](https://jrsoftware.org/ishelp/topic_setup_appname.htm) version [AppVersion](https://jrsoftware.org/ishelp/topic_setup_appversion.htm), localized according to the active language's `NameAndVersion` custom message

So yes, I convinced myself that might do what you claim it should do (but you still need to verify this).

By the way, I would love to see the commit message improved in the following ways:

- Add a first paragraph that explains _why_ you want to change this.
- Include the information I provided above to explain why the code change accomplishes the stated goal (so that not every reader has to go through the same motions as I had to).
- Start the commit message with the prefix "installer:" (there are tons more things in this repository than just the definition of the Git for Windows installer)
- Reading "Specify name of app in list of installed apps" makes me wonder what this is all about, I would have had a really hard time figuring out that this is about the `Add/Remove Programs` list if I had not been part of the conversation, and 3 weeks from now I will have forgotten. Maybe make it explicit by mentioning "Add/Remove Programs" and "Control Panel" in the first line of the commit message?
- A description of the issue, followed by a link to the ticket, would be much appreciated.
- The sign-off should contain a real email address so that you can be reached in case there are follow-up questions.